open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -337,10 +407,28 @@ def get_speech_features_librosa(signal, sample_freq, num_features,
     num_features].
     audio_duration (float): duration of the signal in seconds
   """
-  if augmentation:
-    signal = augment_audio_signal(signal.astype(np.float32), sample_freq, augmentation)
-  else:
-    signal = normalize_signal(signal.astype(np.float32))
+  if augmentation is not None:
+    prob = np.random.rand()
+    if prob < prob_noise[0]:
+      signal = augment_audio_signal(signal, sample_freq, augmentation)
+    prob = np.random.rand()
+    if prob < prob_noise[1]:
+      signal = aug_with_pitch(signal, sample_freq)
+    if custom_noise is not None:
+      if 'noise_level_max' not in augmentation:
```

Comment:
`noise_level_max` and `noise_level_min` are for white noise augmentation. `add_custom_noise` should use other parameters (for example, `custom_noise_level_max`, `custom_noise_level_min`).